### DISCUSSION TITLE
How to access the PostgreSQL DB on the container with SQL client like PgAdmin on host OS?

### DISCUSSION THREAD

**Discussion by flamingotech** [apptension/saas-boilerplate#618] [Q&A]
I tried on Windows 11. Here is what I did:

1. Get the container ID with the command "docker ps."
2. Obtain the IP address using the command "docker inspect <container_id> | grep IPAddress."
3. Connect to the database with PgAdmin on the host OS using the following information:
Hostname: the IP address from step 2
Port: the port obtained from step 1, which is 5432
Database: backend (copied from docker-compose.yml)
Username: backend
Password: backend

I encountered the following error:
"Unable to connect to the server: timeout expired."

Is this the correct way to access the database?
Thank you.

**Comment by mkleszcz** [comment]
Hey,
You don't need to look for internal docker ip. Database container should be accessible on localhost:5432. The port is mapped in the docker-compose.yml file.

**Reply by mh-petcheckin** [reply]
I couldn't get this to work either (running on Debian) using the config above.

I did get it working - all I changed was the host to `host.docker.internal` and I was able to add the server in pgadmin, the rest of the settings (port, username, password) left default as above. I hope this helps someone, and if anyone can patch things so `localhost:5432` would work, even better!
